src/deepsparse/server/config.py

```diff
@@ -121,6 +121,14 @@ class ServerConfig(BaseModel):
     )
 
     endpoints: List[EndpointConfig] = Field(description="The models to serve.")
+    loggers: Union[Dict[str, Dict[str, Any]], str, None] = Field(
```

Comment:
Still think str should not be allowed here. Parsing of this should happen at a different level. Otherwise anything that uses ServerConfig.loggers has to handle parsing the str

Reply:
my thinking here is that if a user has a common logging config on their system they can reference it here. loggers by design already is not useful without being unwrapped by the `logger_manager_from_config` function, ie any use of `ServerConfig.loggers` would need to call into it

Reply:
Okay sure this makes sense then